azure-pipelines.yml

```diff
@@ -0,0 +1,47 @@
+trigger:
+ - master
```

Comment:
Can we make sure this runs on every PR?

Reply:
With this change going in, we'll ensure this will run against every PR.